lib/routes/zju/cse/index.ts

```diff
@@ -0,0 +1,104 @@
+import { Route } from '@/types';
+import cache from '@/utils/cache';
+import got from '@/utils/got';
+import { load } from 'cheerio';
+import { parseDate } from '@/utils/parse-date';
+import timezone from '@/utils/timezone';
+
+export const route: Route = {
+    path: '/cse/:category?',
+    categories: ['university'],
+    example: '/zju/cse/bksjy',
+    parameters: {
+        category: '类别：`bksjy`，默认为简讯专栏，详情在描述中',
+    },
+    features: {
+        requireConfig: false,
+        requirePuppeteer: false,
+        antiCrawler: false,
+        supportBT: false,
+        supportPodcast: false,
+        supportScihub: false,
+    },
+    name: '控制学院通知',
+    maintainers: ['Rabbits-sys'],
+    handler,
+    description: `栏目类型
+
+| 简讯专栏 | 本科生教育 | 研究生教育 | 科研学术 | 人事工作 | 学生思政 | 对外交流 | 就业指导
+| ------ | ------- | ------- | ------ | ------ | ------ | ------ | ------
+|   -    |  bksjy  |  yjsjy  |  kyxs  |  rsgz  |  xssz  |  dwjl  |  jyzd `,
+};
+
+async function handler(ctx) {
+    const category = ctx.req.param('category') ?? ''; // 默认是首页
+    const baseUrl = 'http://www.cse.zju.edu.cn/';
+
+    const categoryMap = {
+        bksjy: { title: '本科生教育', tag: '39322' },
+        yjsjy: { title: '研究生教育', tag: '39333' },
+        kyxs: { title: '科研学术', tag: '39312' },
+        rsgz: { title: '人事工作', tag: '39306' },
+        xssz: { title: '学生思政', tag: '39342' },
+        dwjl: { title: '对外交流', tag: '39353' },
+        jyzd: { title: '就业指导', tag: '39351' },
+    } as const;
+
+    const rootUrl = ['bksjy', 'yjsjy', 'kyxs', 'rsgz', 'xssz', 'dwjl', 'jyzd'].includes(category) ? `${baseUrl}/${categoryMap[category].tag}/list.htm` : `${baseUrl}/39283/list.htm`;
```

Comment:
Use [Object.keys(categoryMap)](https://developer.mozilla.org/docs/Web/JavaScript/Reference/Global_Objects/Object/keys) instead of listing them again.